src/layout/Stack/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './Stack';
+export { Stack as Spacer } from './Stack';
```

Comment:
@amcdnl i think this will work but i'm actually not entirely sure how to fully test this